README.md

```diff
@@ -32,26 +32,34 @@ Whether the selection is by character or by line will also depend on your locati
 
 **Note**: I'm open to adding more queries or support for another language, just open and issue or a PR and I can work with you to get the query working.
 
-## textsubjects-smart
+|                      | `big`                | `container-inner`    | `container-outer`    | `smart`        |
```

Comment:
`big` is actually deprecated, it was the original name for `container-outer, left it in for backwards compatibility then removed it from docs.